cfgov/unprocessed/js/modules/util/web-storage-proxy.js

```diff
@@ -5,25 +5,31 @@
    (local storage and session storage).
    Note: values stored in local storage are not accessible from session storage
    and vice versa. They both work on different objects within the browser.
+   If web storage is not available, values are dumped into an object literal
+   to keep the fuctionality of the API, but will not be saved across sessions.
    ========================================================================= */
 
 'use strict';
 
 // Default storage type.
-var _storage = window.sessionStorage;
+var _storage;
 
 /**
  * Set an item value specified by the key in web storage.
  * @param {string} key The key for the value.
  * @param {string} value The value to store.
- * @param {object} storage (Optional)
+ * @param {Object} storage (Optional)
  *   Use non-persistent storage (sessionStorage)
  *   or persistent storage (localStorage).
  * @returns {string} The value set in web storage.
  */
 function setItem( key, value, storage ) {
   storage = _getStorageType( storage );
-  storage.setItem( key, value );
+  if ( storage.setItem ) {
+    storage.setItem( key, value );
+  } else {
+    storage[key] = value;
```

Comment:
Hmm, not sure how to do that, we would need to throw an error when sessionStorage was called. I added an `xit` and todo in https://github.com/cfpb/cfgov-refresh/pull/1702/commits/c4410225e4822cebaf7f612075c9794c75b788fd